packages/orchestration/src/orchestration-client.test.ts

```diff
@@ -211,7 +195,24 @@ describe('orchestration service client', () => {
         url: 'inference/deployments/1234/completion'
       }
     );
-    const response = await new OrchestrationClient(config).chatCompletion();
+    const response = await new OrchestrationClient(config).chatCompletion({
+      messagesHistory: [
+        {
+          role: 'system',
+          content:
+            'You are a helpful assistant who remembers all details the user shares with you.'
+        },
+        {
+          role: 'user',
+          content: 'Hi! Im Bob'
+        },
+        {
+          role: 'assistant',
+          content:
+            "Hi Bob, nice to meet you! I'm an AI assistant. I'll remember that your name is Bob as we continue our conversation."
+        }
+      ]
+    });
```

Comment:
[req] The test is failing because this prompt needs to be passed to line 188 `constructCompletionPostRequest(config, prompt)` as the second parameter.

Reply:
Fixed the failing test.